nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -106,26 +69,17 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_HOSTNAME = new PropertyDescriptor.Builder()
-            .name("SMTP_HOSTNAME")
```

Comment:
True, so my question is whatever it defaults to is based on the network settings of your machine. Exposing it means I can set it to any name, so I am wondering why would this be important?
Don't get me wrong, putting it back is trivial, but I am looking for justification.